### DISCUSSION TITLE
how to add user

### DISCUSSION THREAD

**Discussion by guidoDimasi** [elsa-workflows/elsa-core#5347] [Q&A] [answered]
a sample for creating a new admin user from postman

**Comment by glime-ai** [comment, marked as answer]
To create a new admin user from Postman for an Elsa Workflow instance, you would typically send a POST request to the appropriate endpoint on the Elsa server with the necessary data. However, based on the context provided, there seems to be an issue with unauthorized access when trying to create a new admin user as mentioned in the GitHub issue from the source `https://github.com/elsa-workflows/elsa-studio/issues/28`.

The request that should be sent from Postman looks like this:

```
POST /elsa/api/identity/applications HTTP/1.1
Host: localhost:5001
Content-Type: application/json

{
    "name": "Postman",
    "roles": ["admin"]
}
```

However, as per the issue reported, this request is returning a 401 Unauthorized error, even though the documentation states that requests from localhost should be allowed.

To resolve this, you should ensure that:

1. The Elsa server is correctly configured to allow requests from localhost.
2. The server is running and accessible at the specified port (in this case, `5001`).
3. Any authentication or authorization configurations that might be blocking the request are correctly set up.

If you are following the documentation and still facing issues, it might be a bug or a misconfiguration that needs to be addressed. In such cases, it's recommended to check the official documentation for any updates or to reach out to the Elsa Workflow support or community for assistance.

Please replace `'localhost:5001'` with the actual URL where your Elsa Workflow instance is running if it's different.

If you need further assistance or clarification, please let me know.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
